### DISCUSSION TITLE
Create App To Use Scratch On Mobile

### DISCUSSION THREAD

**Discussion by z-anonymouz** [ScratchAddons/ScratchAddons#3454] [Discussion]
I think scratch addons team should make an app where it connects to your scratch account then you can use what is basically scratch. I guess like turbo warp editor except you can comment, and share, and it puts those stuff on your scratch account. Comment on this discussion if you think this should happen.

**Comment by retronbv** [comment]
This would be impossible **until** someone has $100 per year and time to develop the app **FOR IOS**

**Reply by cobaltt7** [reply]
Iirc the play store fee is only like 20 dollars one time

The real problem is whether it'd be accepted.

**Comment by WorldLanguages** [comment]
Just use a browser.

**Reply by Dhuls-Scratch** [reply]
> Just use a browser.

This reply should be selected.
Unless you are using a phone/tablet so old, it can only run old Android versions that can only use browsers that somehow aren't supported by Scratch, there is no reason you can't use a browser.

I also think that if anyone were to do this, it should only be the Scratch Team.

**Comment by Samq64** [comment]
An app for Android already exists and Apple rejected the IOS one because it executes code.

**Reply by Dhuls-Scratch** [reply]
iOS one was rejected years ago. Also, it would be allowed nowadays as long as it let the user edit the code.

**Comment by lisa-wolfgang** [comment]
We could make a PWA... that would be pretty slick.

**Reply by Samq64** [reply]
There's already a PWA for TurboWarp. Just use that.
